analytics-core/src/main/java/com/segment/analytics/http/SegmentService.java

```diff
@@ -1,11 +1,12 @@
 package com.segment.analytics.http;
 
 import com.segment.analytics.messages.Batch;
-import retrofit.http.Body;
-import retrofit.http.POST;
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.POST;
 
 /** REST interface for the Segment API. */
 public interface SegmentService {
   @POST("/v1/import")
-  UploadResponse upload(@Body Batch batch);
```

Comment:
This is a breaking change.

Reply:
If there's no way to keep the declaration as `UploadResponse upload(@Body Batch batch);` with the built in functionality, we can use a custom call adapter.